src/components/FeatureList.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'underscore';
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import menuItemPropTypes from './menuItemPropTypes';
+import MenuItem from './MenuItem';
+import styles from '../styles/styles';
+import useLocalize from '../hooks/useLocalize';
+import Text from './Text';
+
+const propTypes = {
+    /** A list of menuItems representing the feature list. */
+    menuItems: PropTypes.arrayOf(PropTypes.shape(menuItemPropTypes)).isRequired,
```

Comment:
@roryabraham `menuItemPropTypes` doesn't have `translationKey` as a type in it.

But we're passing it from LoungeAccessPage. Maybe we should use the `title` key?

Reply:
Good catch

Reply:
sounds good